.circleci/values/hydra.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: oryd/hydra
-  tag: v1.10.3-alpine
+  tag: v1.10.5-alpine
```

Comment:
```suggestion
  # tag: v1.10.5-alpine
```

Reply:
Please comment or delete this part (the whole `image` section), if we update the default version in the chart, then this overrides makes no sense anymore :)

Reply:
yes you are right ;-) changed it!!